### PR TITLE
DCOS-22043: Setup CI

### PR DESCRIPTION
**Pull request by juliangieseke**
This PR adds a simple `Jenkinsfile` including automatic version bumps through `standard-version`. 

I'd like to test if having a new release everytime something merges to master is a good Idea. (This assumes that almost everything is developed in `feature/*` branches and is merged into master once its done. (Which is completely TBD at this point)

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    }
+    stage('Bump Version') {
+      when {
+        expression {
+          release_branches.contains(BRANCH_NAME)
+        }
+      }
+      steps {
+        sshagent (credentials: ['4ff09dce-407b-41d3-847a-9e6609dd91b8']) {
+          sh "git run release"
```
Comment (DanielMSchmidt) [inline Jenkinsfile]: This should be npm run release, right?

Comment (juliangieseke) [reply Jenkinsfile]: yeah 😅 - will fix it when i touch this next time 👍

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+def release_branches = ["master", ] as String[]
+
+pipeline {
+  agent {
+    dockerfile {
+      args  '--shm-size=1g'
+    }
+  }
+  options {
+    timeout(time: 3, unit: 'HOURS')
```
Comment (DanielMSchmidt) [inline Jenkinsfile]: Can we go down to 1 here? Even that is insanely high :)

Comment (juliangieseke) [reply Jenkinsfile]: 15m 👍

**Comment by juliangieseke** [comment]
` DanielMSchmidt approved these changes on behalf of mesosphere/frontend a minute ago ` 🤔 🤩

**Comment by DanielMSchmidt** [comment]
Funny, I wonder where this comes from...

File: Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      steps {
+        sh "npm run test"
+      }
+      post {
+        always {
+          junit 'junit.xml'
+        }
+      }
+    }
+    stage('Bump Version') {
```
Comment (nLight) [inline Jenkinsfile]: So every merge will bump the version? Cool!

**Comment by juliangieseke** [comment]
I requested feedback from the team, you are part of it, so you "resolved" my request by reviewing. 

only bad thing is, that I need a second one :( but we could try to split up our team into <someone has to think about this> :D

**Comment by nLight** [comment]
Sorry, I chose reject when I really wanted to approve 😅

File: Jenkinsfile (hunk 3 above)
Comment (juliangieseke) [reply Jenkinsfile]: yeah, lets try this 👍

**Comment by juliangieseke** [comment]
np, merged. now lets see if `mesosphere-ci` is allowed to push to master 🤞

**Pull request merged**